commcare_connect/opportunity/models.py

```diff
@@ -152,7 +152,9 @@ def claimed_visits(self):
 
     @property
     def approved_visits(self):
-        return CompletedWork.objects.filter(opportunity_access__opportunity=self).count()
+        return CompletedWork.objects.filter(
+            opportunity_access__opportunity=self, status=CompletedWorkStatus.approved
```

Comment:
is this change intentional?

Reply:
Yeah, this was found in the QA for this ticket where the Approved VIsits counts showed all visits.